### ISSUE TITLE
Support passing list of locations to the grid-intensity-exporter

### ISSUE THREAD

**Issue by ZeidH** [thegreenwebfoundation/grid-intensity-go#80]
Hey! 

We're using the grid-intensity-exporter in our Kubernetes cluster to get the Carbon Average data into Prometheus. It's working really well, however, we'd like to see the Grid Intensity of multiple locations.

Currently the `GRID_INTENSITY_LOCATION` seem to only support a single location. To get metrics of multiple locations we'd need to deploy several Deployments. This gets especially tricky with when these deployments need to be within the same namespace.

**Comment by mrchrisadams** [comment]
Hi @ZeidH - I think this is actually quite a good idea - can you let us know which proveder you're using for this? 

I think one reason we haven't done this before was that when connecting to a third party provider, you can only use one location on free plans.

Of course that doesn't apply for all of them - we could totally do this with the default Ember provider, for example.

I think the hard part might be figuring out an elegant way to expose multiple locations from the same provider in the promethus exporter syntax.

WDYT @rossf7 ? I don't know the idiomatic way multiple instances of a set of metrics are exposed by prometheus exporters, and I'm not sure where to look.

**Comment by rossf7** [comment]
Hi @ZeidH, I agree this is a good idea and better than running multiple deployments.

> I think the hard part might be figuring out an elegant way to expose multiple locations from the same provider in the promethus exporter syntax.

@mrchrisadams we should be OK because the prometheus metrics have a location label and we can expose metrics per location. We do something similar for WattTime because we expose both relative and actual values for carbon intensity.

The challenge I see is how we accept the input. One solution would be to extend the `--location` param to accept a CSV list of locations. WDYT?

This would also work with the CLI because the output is a JSON array. Maybe this is why we made it an array? I can't remember it was a while ago! :)

**Comment by ZeidH** [comment]
Another solution could be to extend the `exporter.go` to iterate over a CSV list when collecting, this would also work for providers that don't allow a location list.

**Comment by ZeidH** [comment]
Had some spare time to work on this, I've made a high-level implementation that extends both the exporter and cli to support CSV input: #82 
This is my first time working with Go so feedback is appreciated :D 

There are a few things that I've noticed that I'd like your opinions on:
- The input param name is in singular (location). WDYT we should change this into? I was thinking of just leaving it as is, with just clarifying the CSV ability in the description.
- As it is a high-level implementation, I saw that Rate limiting may become an issue for some providers as it iteratively just queries one location from the provider. Is this a non-issue, WDYT?
